src/controllers/account.controller.js

```diff
@@ -27,13 +33,14 @@ export default {
 * @param {string} res
 * @returns {object} returns the transaction details if succesful
 */
-  show(req, res) {
+  async show(req, res) {
```

Comment:
rename this function, maybe "getDetail" or whatever

Reply:
Noted